esp/esp/program/models/__init__.py

```diff
@@ -1865,6 +1865,22 @@ def __unicode__(self):
         else:
             return self.name
 
+class PhaseZeroRecord(models.Model):
+
+    class Meta:
```

Comment:
Yeah, I meant class Meta. This is one of those cases where the older code isn't a good example--it was needed on older versions of Django but isn't anymore, and we didn't bother deleting all the uses in this file when we updated.